mmv1/third_party/terraform/resources/resource_bigtable_table.go

```diff
@@ -109,6 +119,15 @@ func resourceBigtableTableCreate(d *schema.ResourceData, meta interface{}) error
 	tableId := d.Get("name").(string)
 	tblConf := bigtable.TableConf{TableID: tableId}
 
+	// DeletionProtection can be protected or unprotected
+	// Check if deletion protection is given
+	deletionProtection := d.Get("deletion_protection")
+	if deletionProtection == "PROTECTED" {
+		tblConf.DeletionProtection = bigtable.Protected
+	} else if deletionProtection == "UNPROTECTED" {
+		tblConf.DeletionProtection = bigtable.Unprotected
+	}
```

Comment:
Add comment here: If not set, d.Get("deletion_protection") will return empty and the value will be set to the API default.